SystemInfo/Collector/DatabaseSystemInfoCollector.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * File containing the DatabaseSystemInfoCollector class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzSupportToolsBundle\SystemInfo\Collector;
+
+use Doctrine\DBAL\Connection;
+use EzSystems\EzSupportToolsBundle\SystemInfo\Value;
+
+/**
+ * Collects database information.
+ */
+class DatabaseSystemInfoCollector implements SystemInfoCollector
```

Comment:
The implementation would actually be a `DoctrineDatabaseSystemInfoCollector`, would it not ?